template/docs/conf.py.jinja

```diff
@@ -109,8 +110,11 @@ master_doc = 'index'
 # built documents.
 #
 
-release = get_version("{{projectname}}")
-version = ".".join(release.split('.')[:3])  # CalVer
+try:
+    release = get_version("{{projectname}}")
+    version = ".".join(release.split('.')[:3])  # CalVer
+except PackageNotFoundError:
+    release = version = "0.0.0-dev"
```

Comment:
Can you add a print or log to make it easier to trace if something breaks in CI? Otherwise :+1: